public/static/partials/patch_trigger_alias_modal.html

```diff
@@ -30,15 +30,42 @@
                         </md-select>
                     </md-input-container>
                 </section>
-                <section layout="row" flex>
-                    <md-input-container flex=50>
-                        <label>Variant regex:</label>
-                        <input type="text" ng-model="alias.variant_regex" placeholder="select variants matching regex">
-                    </md-input-container>
-                    <md-input-container flex=50>
-                        <label>Task regex:</label>
-                        <input type="text" ng-model="alias.task_regex" placeholder="select tasks matching regex">
-                    </md-input-container>
+                <section layout="column" flex>
+                    <section layout="row" flex ng-repeat="(index, specifier) in alias.task_specifiers">
+                        <md-input-container flex=33>
+                            <input type="text" ng-model="specifier.variant_regex" placeholder="variant regex">
+                        </md-input-container>
+                        <md-input-container  flex=33>
+                            <input type="text" ng-model="specifier.task_regex" placeholder="task regex">
+                        </md-input-container>
+                        <md-input-container  flex>
+                            <input type="text" ng-model="specifier.patch_alias" placeholder="patch alias">
+                        </md-input-container>
+                        <div>
+                            <button class="btn btn-default btn-danger" type="button"
+                                style="float: left" ng-click="removeTaskSpecifier(index)">
+                                <i class="fa fa-trash"></i>
+                            </button>
+                        </div>
+                    </section>
+                    <section layout="row" flex>
+                        <md-input-container  flex=33>
+                            <input type="text" ng-model="new_task_specifier.variant_regex" placeholder="variant regex">
```

Comment:
Can you add a description here to say "either patch name OR variant and task"?

Reply:
done
the exact wording I chose was "One of patch alias and variant/task regex set"